lib/src/index.ts

```diff
@@ -180,3 +180,8 @@ export { default as NxFieldset, Props as NxFieldsetProps } from './components/Nx
 export { default as NxToggle, Props as NxToggleProps } from './components/NxToggle/NxToggle';
 export { default as NxStatefulToggle, Props as NxStatefulToggleProps }
   from './components/NxToggle/stateful/NxStatefulToggle';
+
+export { default as NxTag, NxSelectableTag, Props as NxTagProps }
```

Comment:
Should export `NxSelectableTagProps` as well

Reply:
> Should export `NxSelectableTagProps` as well

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/690d730a479ef678e4c71fbf3984187c6dbd50c4